third-party/CMakeLists.txt

```diff
@@ -37,3 +37,19 @@ if(THEROCK_BUNDLE_SYSDEPS)
     add_subdirectory(sysdeps/windows)
   endif()
 endif()
+
+# gRPC: Static library dependency for RDC
+# - Only built when RDC is enabled (Linux only)
+# - Depends on bundled zlib from sysdeps (via RUNTIME_DEPS)
+# - See: docs/rfcs/RFC0007-rdc-therock-integration.md
+if(THEROCK_ENABLE_RDC)
```

Comment:
Don't need to gate this or do any other checks. If nothing depends on grpc, it won't be fetched/built and better to include it in the build vs filtering. Also, the build files should work on Windows as they might get extended there in the future (so shouldn't hard gate).